emscripten-releases-tags.json

```diff
@@ -10,6 +10,8 @@
     "latest-releases-upstream": "latest"
   },
   "releases": {
+    "3.1.25": "ff6babb041d0f31575cc16d15ef82c6222ca99b8",
+    "3.1.25-asserts": "6b19d6a8c30d7b83ba2193625fc12cce9ae0206b",
```

Comment:
There are more differences than I would expect in the emscripten-releases repo between these two hashes. Why do they get different versions of binaryen, llvm, etc?